typescript/framework-extensions/vercel-ai-sdk/package.json

```diff
@@ -41,8 +41,13 @@
     "ai-sdk"
   ],
   "dependencies": {
-    "@coinbase/agentkit": "^0.2.3",
-    "ai": "^4.1.16",
     "zod": "^3.22.4"
+  },
+  "devDependencies": {
+    "@coinbase/agentkit": "0.2.3"
+  },
```

Comment:
This is a common pattern when depending on a package as a peer dependency, but still needing an installed copy for building / testing. For example, see wagmi's [package.json](https://github.com/wevm/wagmi/blob/main/packages/react/package.json#L81-L106).

An additional detail here is that `@coinbase/agentkit` is a package in the monorepo, so the version number should match up with the version in that package, otherwise npm will try to install and use a published version, rather than the local one. This is handled more gracefully by [pnpm workspaces](https://pnpm.io/workspaces)